setup.py

```diff
@@ -88,6 +88,7 @@
         "django>=1.7",
         "pytz",
         "six",
+        "django-ipware==1.1.6"
```

Comment:
django-ipware is only used for the `InitialManagedAccountForm` - might be better to make this an optional dependency then only (in case it cannot be avoided altogether).

Reply:
@blueyed I've had a think about this - it strikes me that having an optional dependency creates complexity during deployment.

If you have a project that uses pinax-stripe and you are using InitialManagedAccountForm and installing pinax-stripe doesn't include django-ipware, doesn't that put the deployer in an odd situation where they have to remember to install django-ipware as part of their projects specific dependencies?

Let me know if you have a good reference implementation of an optional dependency in Python that I can take a peek at.

Reply:
You can use `extras_require` for this, see e.g. https://github.com/celery/celery/blob/de2d075fb07850eee247766ebeb11d5530a43734/setup.py#L217 for something more involved.

Reply:
There could then be e.g. `pinax-stripe[forms]` (to have all dependencies for forms) and/or `pinax-stripe[full]` (to really have all optional deps).

For now `pinax-stripe[ipware]` could be used - but I am not sure how much hassle it is for when you expect that everything works out of the box?!

Reply:
Neat that there is a mechanism for this.

If `django-ipware` was a heavyweight dependency with serious storage, memory or cascading dependency consequences it would seem worthwhile to me to go that route. As it stands, `django-ipware` is a lightweight, pure-python lib with no dependencies. My gut instinct is that it is not a strong candidate for use of `extras_require`.

Reply:
> is a lightweight

Even though it adds up in your requirements - and I assume thata lot of users of this lib are not using forms (and even this specific) one.

Reply:
@blueyed once we're merged, feel free to knock up a PR and we can discuss with @paltman. My sense is that adding complexity to avoid a tiny dependency is not the way to go so I'm not keen on spending time further time on this.